corpus/templates/accounts/login.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'base.html' %}
+
+{% block title %}
+Login
+{% endblock %}
+
+{% block content %}
+<div class="card w-1/2 mx-auto my-10 bg-base-100 glass shadow-xl prose">
```

Comment:
Not really a fan of the glass theme here as well, I think you can take reference from popular website login pages (LeetCode is a good example) to style the page better.